fairlearn/metrics/_extra_metrics.py

```diff
@@ -6,38 +6,175 @@
 These are metrics which are not part of `scikit-learn`.
 """
 
+import numpy as np
 import sklearn.metrics as skm
 
 from ._balanced_root_mean_squared_error import _balanced_root_mean_squared_error  # noqa: F401
 from ._mean_predictions import mean_prediction, _mean_overprediction, _mean_underprediction  # noqa: F401,E501
 from ._selection_rate import selection_rate  # noqa: F401,E501
 
+_NEED_TWO_UNIQUE_Y_VALS = "Must have two unique y values"
+_NEED_POS_LABEL_IN_Y_VALS = "Must have pos_label in y values"
 
-def true_positive_rate(y_true, y_pred, sample_weight=None):
-    r"""Calculate the true positive rate (also called sensitivity, recall, or hit rate)."""
+
+def _get_labels_for_confusion_matrix(y_true, y_pred, pos_label):
+    r"""Figure out the labels argument for skm.confusion_matrix.
+
+    This is an internal method used by the true/false positive/negative
+    rate metrics (and hence are restricted to binary data). We compute
+    these using the confusion matrix.
+    In general, we expect the input data to be :math:`\in {0, 1}`
```

Comment:
no need for :math: here; just `to be in {0, 1}`

Reply:
I think we've gone for the LaTeX rendering of these in other places.

Reply:
The issue is not with the webpage, but with calling `help` in python. Also, as a matter of style, I think that we should only resort to math symbols if they make things clearer. I don't think it's the case here.

Reply:
Done.